keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -85,22 +96,30 @@ class ClusterHandler(
       resourceDiff
         .toIndividualDiffs()
         .filter { diff -> diff.hasChanges() }
-        .map { diff ->
+        .mapNotNull { diff ->
           val desired = diff.desired
           val job = when {
-            diff.isCapacityOnly() -> diff.resizeServerGroupJob()
-            else -> diff.createServerGroupJob() + resource.spec.deployWith.toOrcaJobProperties()
+            diff.isCapacityOnly() -> listOf(diff.resizeServerGroupJob())
+            diff.isAutoScalingOnly() -> diff.modifyScalingPolicyJob()
```

Comment:
should this be `listOf(diff.modifyScalingPolicyJob())` like the other ones around it?

Reply:
`modifyScalingPolicyJob()` returns a list, it's designed to generate multi-stage orchestrations.

Reply:
Ahh gotcha. Thanks!